to_do_item.py

```diff
@@ -0,0 +1,14 @@
+class ToDoItem:
+    def __init__(self, id, title, status):
+        self.id = id
+        self.title = title
+        self.status = status
+
+    @staticmethod

```

Comment:
These 2 methods should arguably be `@classmethod`s to ovoid repeating the class name.